### DISCUSSION TITLE
[Docs] [input] Please add the "wrap" attribute to el-input if type="textarea"

### DISCUSSION THREAD

**Discussion by sagid-veliev** [element-plus/element-plus#11620] [Feature Request]
Feature type: **`Docs`**

## Details

- Enhanced component name: `input`

## Description

Please add the "wrap" attribute to el-input if type="textarea". A case may arise when it is necessary to save the value from textarea together with the hyphenation signs like "/n".

### Additional comments

_(empty)_

**Comment by btea** [comment]
Hmm, I don't understand what is your purpose of adding this attribute? Can you explain it?
